Python 3 uses gdbm by default, but we explicitly enable ndbm, so fix build script to create ndbm DB
--- avahi-0.7/service-type-database/build-db.1	2020-01-20 12:58:41.232303391 +0000
+++ avahi-0.7/service-type-database/build-db	2020-01-20 12:59:04.296418239 +0000
@@ -20,7 +20,7 @@
 try:
     import anydbm as dbm
 except ImportError:
-    import dbm
+    import dbm.ndbm as dbm
 
 import sys
 
